### PR TITLE
Fix direct routing & disable cache

### PR DESCRIPTION
**Pull request by mdomasevicius**
- Disable cache, we don't need it right now
- Use hash router to relive 404 on direct routes

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,17 +1,49 @@
...
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta
+        name="description"
+        content="Web site created using create-react-app"
+    />
+    <meta http-equiv="Pragma" content="no-cache"/>
+    <meta http-equiv="Expires" content="0"/>
+    <link rel="shortcut icon" href="/favicon.ico">
+    <title>Mysterium</title>
```
Comment (Waldz) [inline public/index.html:13]: Mysterium Node UI

**Review by Waldz — APPROVED** [review]

File: public/index.html [hunk 2]
```diff
@@ -1,17 +1,49 @@
...
-  <link rel="shortcut icon" href="/favicon.ico">
-  <title>Mysterium</title>
-  <script>
-    (function(){var w=window;var ic=w.Intercom;if(typeof ic==="function"){ic('reattach_activator');ic('update',w.intercomSettings);}else{var d=document;var i=function(){i.c(arguments);};i.q=[];i.c=function(args){i.q.push(args);};w.Intercom=i;var l=function(){var s=d.createElement('script');s.type='text/javascript';s.async=true;s.src='https://widget.intercom.io/widget/h7hlm9on';var x=d.getElementsByTagName('script')[0];x.parentNode.insertBefore(s,x);};if(w.attachEvent){w.attachEvent('onload',l);}else{w.addEventListener('load',l,false);}}})();
-  </script>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta
+        name="description"
+        content="Web site created using create-react-app"
```
Comment (chompomonim) [inline public/index.html:8]: Maybe some better description?

File: public/index.html [hunk 3]
```diff
@@ -1,17 +1,49 @@
...
+    />
+    <meta http-equiv="Pragma" content="no-cache"/>
+    <meta http-equiv="Expires" content="0"/>
+    <link rel="shortcut icon" href="/favicon.ico">
+    <title>Mysterium</title>
+    <script>
```
Comment (chompomonim) [inline public/index.html:14]: Why not leave this script as one liner?

**Review by chompomonim — APPROVED** [review]

File: public/index.html (hunk 3 above)
Comment (mdomasevicius) [reply public/index.html:14]: autoformat

**Pull request merged**